### PR TITLE
small tweaks for bangle dev editor

### PR DESCRIPTION
**Pull request by mattcasey**
Some cleanup items/bug fixes:
- Simplified CSS for page title and controls, reducing the need to set multiple 'top' offsets - now we just have an offset whether there's an icon set or not.
- made placeholder  its own component, and now it appears in the correct spot wherever it is used
- the text style cursor was disabled when looking at editor inside focalboard

### CODE REVIEW

File: components/common/Emoji.tsx [hunk 1]
```diff
@@ -11,10 +11,9 @@ const Emoji = styled(Box)`
   white-space: nowrap;
   user-select: none;
   cursor: pointer;
-  padding: ${({ theme }) => theme.spacing(1)};
   border-radius: ${({ theme }) => theme.spacing(0.5)};
-  width: 100px;
-  height: 100px;
+  width: 80px;
```
Comment (mattcasey) [inline components/common/Emoji.tsx:15]: less padding around icons

File: components/editor/Placeholder.tsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import { useMemo } from 'react';
```
Comment (mattcasey) [inline components/editor/Placeholder.tsx:1]: moved placeholder to its own file

File: theme/@bangle.dev/styles.scss [hunk 3]
```diff
@@ -23,10 +24,14 @@
 }
 
 :root {
-  --window-tooltip-zIndex: 5;
+  --window-tooltip-zIndex: 4;
   --inlineUniversalPalette-height: 400px;
 }
 
+.bangle-tooltip {
```
Comment (mattcasey) [inline theme/@bangle.dev/styles.scss:31]: fixes a bug where the tooltip apears under the top header

**Comment by Devorein** [comment]
One small thing I noticed is that the button for adding Page Cover is overlapping with the Page Icon itself
image

**Comment by Devorein** [comment]
I've also noticed that the placeholder issue still exist for the Bounty form unfortunately 😦 
image

**Pull request merged**